vision/unit_tests/test_annotations.py

```diff
@@ -67,17 +67,26 @@ def test_ctor(self):
         self.assertEqual(annotations.texts, [True])
 
     def test_from_pb(self):
+        from google.cloud.vision.likelihood import Likelihood
         from google.cloud.grpc.vision.v1 import image_annotator_pb2
 
         image_response = image_annotator_pb2.AnnotateImageResponse()
         annotations = self._make_one().from_pb(image_response)
         self.assertEqual(annotations.labels, [])
         self.assertEqual(annotations.logos, [])
-        self.assertEqual(annotations.faces, ())
+        self.assertEqual(annotations.faces, [])
         self.assertEqual(annotations.landmarks, [])
         self.assertEqual(annotations.texts, [])
-        self.assertEqual(annotations.safe_searches, ())
-        self.assertEqual(annotations.properties, ())
+        self.assertEqual(len(annotations.safe_searches), 1)
```

Comment:
On the other protobuf objects, there has been a way for me to tell if there are results or not. In this case, the values for `adult`, `spoof` etc... are defaulting to`0` which is the valid `Likelihood.UNKNOWN` type.

I would have to track the feature type's that are passed in from the top in order to determine if this was a requested feature type or not. There has to be a better way.